### PR TITLE
Update distribute-offline-apps.md

### LINKED ISSUE #8942: Broken link - Distribute offline apps
Hi all.

In Distribute offline apps (https://docs.microsoft.com/en-us/microsoft-store/distribute-offline-apps) the link Manage apps from Microsoft Store for Business with Microsoft Intune that points to https://docs.microsoft.com/en-us/intune/deploy-use/manage-apps-you-purchased-from-the-windows-store-for-business-with-microsoft-intune rports error 404. Probably the destination has changed URL. I believe that it should point to the doc How to manage volume purchased apps from the Microsoft Store for Business with Microsoft Intune - https://docs.microsoft.com/en-us/mem/intune/apps/windows-store-for-business.

Can you double check this and fix it please?

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

From issue ticket #8942 (**Broken link - Distribute offline apps**):

> Hi all.
> 
> In Distribute offline apps (https://docs.microsoft.com/en-us/microsoft-store/distribute-offline-apps) the link Manage apps from Microsoft Store for Business with Microsoft Intune that points to [docs.microsoft.com/en-us/intune/deploy-use/manage-apps-you-purchased-from-the-windows-store-for-business-with-microsoft-intune](https://docs.microsoft.com/en-us/intune/deploy-use/manage-apps-you-purchased-from-the-windows-store-for-business-with-microsoft-intune) rports error 404. Probably the destination has changed URL. I believe that it should point to the doc How to manage volume purchased apps from the Microsoft Store for Business with Microsoft Intune - [docs.microsoft.com/en-us/mem/intune/apps/windows-store-for-business](https://docs.microsoft.com/en-us/mem/intune/apps/windows-store-for-business).
> 
> Can you double check this and fix it please?

Thanks to @joaonltome for noticing and reporting this broken link and also suggesting a likely solution.

**Changes proposed:**

- Replace the broken link /microsoft-store/distribute-offline-apps with the working page link /mem/intune/apps/windows-store-for-business .
- ~~Change the paragraph title "To download an offline-licensed app" from **bold** formatting to a H3 heading, as well as including the HTML anchor in that heading.
  (This heading level might possibly need to be a H4 heading size to remain the same size as its current **bold**-only format.)~~

**Whitespace changes:**

- Remove 13 redundant blank lines at the end of the document page.
- By removing these 13 blank lines, we also remove that redundant empty command copy box (created by indents).
- Remove the Line break (NewLine) between the anchor link and the paragraph title, to see if the common format works.
- Reduce 12 occurrences of 3 blank spaces after bullet point list indicators to 1 space.
- Reduce 6 occurrences of double blank space after numbered list indicators to 1 space.
- Add missing colon in **Applies to:**
 
**Ticket closure or reference:**

Closes #8942

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
- **Regular source view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8950/files
- **Hide whitespace changes:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8950/files?diff=unified&w=1 (focuses on visible or effective changes)
- **Rich Diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8950/files?diff=unified&w=1&short_path=5b7d86f (mostly useful to evaluate if H3 or H4 should be used in the last section)

**Comment by illfated** [comment]
@JohanFreelancer9 : copy review, please (if & when able).

**Comment by illfated** [comment]
Also, to all reviewers: Please check if we should include any additional changes in the link description text, to match the page titles exactly, or if it is preferred to keep the current text as is.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Hmm, I can't seem to enable the anchor link functionality to make it appear in the Rich Diff view, neither in this PR nor in the file preview in my branch. Oh well, if it doesn't work on the currently existing page, it should not make much difference. The new change adheres to the most common anchor link usage in other documents.

File: store-for-business/distribute-offline-apps.md [hunk 1]
```diff
@@ -18,58 +18,57 @@ ms.date: 10/17/2017
...
 
 ## Why offline-licensed apps?
 
 Offline-licensed apps offer an alternative to online apps, and provide additional deployment options. Some reasons to use offline-licensed apps:
 
--   **You don't have access to Microsoft Store services** - If your employees don't have access to the internet and Microsoft Store services, downloading offline-licensed apps and deploying them with imaging is an alternative to online-licensed apps.
+- **You don't have access to Microsoft Store services** - If your employees don't have access to the internet and Microsoft Store services, downloading offline-licensed apps and deploying them with imaging is an alternative to online-licensed apps.
```
Comment (JohanFreelancer9) [inline store-for-business/distribute-offline-apps.md:32]:
```suggestion
- **You don't have access to Microsoft Store services** - If your employees don't have access to the Internet and Microsoft Store services, downloading offline-licensed apps and deploying them with imaging is an alternative to online-licensed apps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - just one more additional minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**